NuGet.Docs/Release-Notes/NuGet-3.4-RC.md

```diff
@@ -0,0 +1,38 @@
+
+# NuGet 3.4-RC Release Notes
+
+[NuGet 3.3 Release Notes](nuget-3.3) 
+
+NuGet 3.4-RC was released March 3, 2016 alongside the Visual Studio 2015 Update 2 RC and was built with a few tenets in minds:
+
+*  Cross-Platform support
+*  Performance improvements
+*  Minor UI improvements
+
+## New Features
+
+* NuGet clients now support gzip content-encoding from repositories
+* Support for PDBs from packages in xproj projects
+* Support for iOS and Android build actions in the contentFiles element
+* Support for the netstandard and netstandardapp framework monikers
+
+## New User Interface Features
+
+* Significant performance improvements especially on the Installed, Updates, and Consolidate tabs
+* Installed and Updates tabs are now sorted alphabetically
+* Added a Refresh button that allows a search to be refreshed
+* 'All' package sources search has been reintroduced with improved support for slower package repositories
```

Comment:
All of the stuff here is RTM only. 

All sources should get a higher visibility on top